frappe/website/doctype/website_settings/website_settings.js

```diff
@@ -10,7 +10,7 @@ frappe.ui.form.on('Website Settings', {
 			frappe.msgprint(__("Select a Brand Image first."));
 		}
 		frm.set_value("brand_html", "<img src='"+ frm.doc.banner_image
-			+"' style='max-width: 150px;'>");
+			+"' style='max-width: 150px; max-height: 25px;'>");
```

Comment:
No inline styles, please. If old code uses inline styles, fix it by adding a class instead.